[2024-12-17T10:39:00.832+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-17T10:39:00.889+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Our_first_dag_using_python_operator.greet scheduled__2024-12-17T00:13:00+00:00 [queued]>
[2024-12-17T10:39:00.910+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Our_first_dag_using_python_operator.greet scheduled__2024-12-17T00:13:00+00:00 [queued]>
[2024-12-17T10:39:00.912+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-12-17T10:39:00.975+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): greet> on 2024-12-17 00:13:00+00:00
[2024-12-17T10:39:00.992+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2597) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-17T10:39:00.995+0000] {standard_task_runner.py:72} INFO - Started process 2610 to run task
[2024-12-17T10:39:01.005+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'Our_first_dag_using_python_operator', 'greet', 'scheduled__2024-12-17T00:13:00+00:00', '--job-id', '303', '--raw', '--subdir', 'DAGS_FOLDER/create_dag_with_python.py', '--cfg-path', '/tmp/tmpv6dj8phe']
[2024-12-17T10:39:01.009+0000] {standard_task_runner.py:105} INFO - Job 303: Subtask greet
[2024-12-17T10:39:01.115+0000] {task_command.py:467} INFO - Running <TaskInstance: Our_first_dag_using_python_operator.greet scheduled__2024-12-17T00:13:00+00:00 [running]> on host 8348eb091483
[2024-12-17T10:39:01.323+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='srush' AIRFLOW_CTX_DAG_ID='Our_first_dag_using_python_operator' AIRFLOW_CTX_TASK_ID='greet' AIRFLOW_CTX_EXECUTION_DATE='2024-12-17T00:13:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-17T00:13:00+00:00'
[2024-12-17T10:39:01.325+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-17T10:39:01.356+0000] {logging_mixin.py:190} INFO - Hello World!
[2024-12-17T10:39:01.357+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-17T10:39:01.378+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-17T10:39:01.379+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=Our_first_dag_using_python_operator, task_id=greet, run_id=scheduled__2024-12-17T00:13:00+00:00, execution_date=20241217T001300, start_date=20241217T103900, end_date=20241217T103901
[2024-12-17T10:39:01.467+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-17T10:39:01.498+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-17T10:39:01.504+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
